plugins/talk-plugin-featured-comments/client/components/FeaturedTag.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styles from './FeaturedTag.css';
+import {t} from 'plugin-api/beta/client/services';
+
+const isFeatured = (tags) => !!tags.filter((t) => t.tag.name === 'FEATURED').length;
```

Comment:
The double !! terrifies me ha. But agreed, `some` is the better function we're looking for here.